src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquisition.cs

```diff
@@ -641,7 +647,7 @@ private IConfidentialClientApplication BuildConfidentialClientApplication(Merged
                     else
                     {
                         string? sessionKey = tokenAcquisitionOptions!.LongRunningWebApiSessionKey;
-                        if (sessionKey == TokenAcquisitionOptions.LongRunningWebApiSessionKeyAuto)
+                        if (sessionKey == Abstractions.AcquireTokenOptions.LongRunningWebApiSessionKeyAuto)
```

Comment:
Is it something we missed?

Reply:
i don't think so, it was a suggestion from VS